### PR TITLE
321/process nwb2

### PR DESCRIPTION
**Pull request by sgratiy**
Address #321

### CODE REVIEW

**Comment by wbwakeman** [comment]
If you are interested, I have hundreds of MIES generated NWB2 files at /allen/aibs/technology/waynew/ephys/output/Z/ephys/20200207_nwb2/

File: ipfx/hbg_dataset.py [hunk 1]
```diff
@@ -93,3 +86,15 @@ def get_stimulus_code_ext(self, sweep_num):
 
     def get_recording_date(self):
         return self.nwb_data.get_recording_date()
+
+def get_finite_or_none(d, key):
+
+    try:
+        value = d[key]
+    except KeyError:
```
Comment (MatthewAitken) [inline ipfx/hbg_dataset.py]: `d.get(key)` is imo a much more succinct and explicit way to do this- get() returns none by default, or you can specify a default value, e.g. `d.get(key, None)`. It is very slightly slower though, so if this is done a whole lot maybe just leave it.

File: ipfx/nwb_reader.py [hunk 2]
```diff
@@ -111,17 +113,17 @@ def get_unit_name(stim_attrs):
     def get_long_unit_name(unit):
         if not unit:
             return "Unknown"
-        if unit.startswith('A'):
+        elif unit.startswith('A') or unit == 'amps':
```
Comment (MatthewAitken) [inline ipfx/nwb_reader.py]: is this as specific of a check we need? `startswith` seems possibly ambigious...

Comment (sgratiy) [reply ipfx/nwb_reader.py]: I specified unit names explicitly as you suggested. thanks.

**Pull request merged**